roles/pip/defaults/main.yaml

```diff
@@ -0,0 +1,3 @@
+---
+# Version of pip to be installed
+pip_version: 18.0
```

Comment:
I noticed that we are using a number instead of a string for the version here. I tested it and it seems like Ansible is smart enough to convert this to a string without removing training 0 (e.g. 18.0.0 will be correctly converted to "18.0.0"), but I think it's still better to add quotes explicitly like here: https://github.sig.oregonstate.edu/ecs-data/ansible-private-roles/blob/master/roles/jenkins/vars/main.yml just in case. What do you think?

I also noticed there are lots of same issues like https://github.com/osu-mist/ansible-roles/blob/master/roles/docker/defaults/main.yml#L4 and https://github.sig.oregonstate.edu/ecs-data/ansible-private-roles/blob/master/common_vars.yml#L29-L30 If you agree, could you go through our other roles as well?

Reply:
Sounds good. I think I looked at some random role to see how versions were specified and it must have been one that used a float. Makes sense to use a string though. I'll go ahead and change that for this role as well as the others in ansible-roles. I'll change the versions for ansible-private roles in my other PR for this ticket (ECSOPS-204).